auth-api/migrations/versions/031a07fb0811_add_account_id_column_in_tasks_table.py

```diff
@@ -20,13 +22,72 @@
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.add_column('tasks', sa.Column('account_id', sa.Integer(), nullable=True))
-    op.add_column('tasks', sa.Column('relationship_status', sa.String(length=100), nullable=True,
-                                     server_default=TaskRelationshipStatus.PENDING_STAFF_REVIEW.value))
+    op.add_column('tasks', sa.Column('relationship_status', sa.String(length=100), nullable=True))
+
+    conn = op.get_bind()
+    org_res = conn.execute(f"SELECT * FROM orgs WHERE status_code in ('PENDING_STAFF_REVIEW', 'REJECTED', 'APPROVED');")
```

Comment:
Do we even have 'APPROVED" status for orgs?

Reply:
updated :)